kernel/patches-4.11.x/README.md

```diff
@@ -0,0 +1,82 @@
+# How to build 4.11.x based custom Linux kernel for LCOW
+
+You can download the Linux 4.11 source code from [kernel.org](https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.11.tar.xz).
+
+Once you get the _4.11 kernel_, apply all the following patches 
+
+## 1. Patch for "nvdimm: Lower minimum PMEM size"
+
+This [patch file](./0002-NVDIMM-reducded-ND_MIN_NAMESPACE_SIZE-from-4MB-to-4K.patch) could be find in this directory.  
+
+You should be in the Linux kernel source directory before applying the patch with the following command
+
+```
+patch -p1 < /path/to/kernel/patches-4.11.x/0002-*
+```
+
+## 2. Patch set for the Hyper-V vsock support
+
+These patches enables the **Hyper-V vsock transport** feature,
+this instructions is to get them from a developer repository and
+assuming you have a _Linux GIT repository_  already
+
+```
+git config --global user.name "yourname"
+git config --global user.email youremailaddress 
+ 
+git remote add -f dexuan-github https://github.com/dcui/linux.git
+ 
+git cherry-pick c248b14174e1337c1461f9b13a573ad90a136e1c
```

Comment:
I would recommend adding a `-x` to all `git cherry-pick` lines as this keeps the original hash in the commit message.

Reply:
I didn't know that useful option. I will add that.